gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
Do we **need** Gradle 8.7 for this? If we don't, I'd prefer to make this into a different change.

Reply:
I'll rerun the tests with Gradle 8.6 and check if it works. I did upgrade to 8.7 for a reason, there was something that had been fixed, but I can do that separately if the branch as it currently is passes the tests anyway.